### PR TITLE
qm-disk-resize: add page; qm-resize: add alias page

### PR DESCRIPTION
**Pull request by BabaYaga1221**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> I would suggest merging this branch with the main as the `qm-suspend` page was merged a couple of days ago. In future, when creating PRs I would suggest creating a fresh branch with the head branch as `tldr-pages/tldr:main`. An alternative is enabling allow maintainers to write to this branch in the right side of page so that we could do the same for you.

**Review by kbdharun — CHANGES_REQUESTED** [review]
Just now checked the man page again, I think this page can be made into an [alias page](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md#en) and the other commands be moved to the `qm-disk-resize` page.

File: pages/linux/qm-disk-resize.md
Comment (kbdharun) [inline pages/linux/qm-disk-resize.md:8]:
```suggestion
# qm disk resize

> Resize a virtual machine (VM) in the Proxmox Virtual Environment (PVE).
> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

- Resize a virtual machine by id:

`qm disk resize {{vm_id}} {{disk_name}} +{{resize_value}}G`
```

File: pages/linux/qm-resize.md
Comment (kbdharun) [inline pages/linux/qm-resize.md:8]:
```suggestion
# qm resize

> This command is an alias of `qm-disk-resize`.
> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

- View documentation for the original command:

`tldr qm-disk-resize`
```

**Review by kbdharun — APPROVED** [review]

**Comment by BabaYaga1221** [comment]
Thanks @kbdharun , can you merge this work?

**Comment by kbdharun** [comment]
> can you merge this work?

This PR needs one more approval review from other maintainers to be merged.

File: pages/linux/qm-disk-resize.md
Comment (acuteenvy) [inline pages/linux/qm-disk-resize.md:8]:
```suggestion
- Add `n` gigabytes to a virtual disk:

`qm disk resize {{vm_id}} {{disk_name}} +{{n}}G`
```

File: pages/linux/qm-disk-resize.md
Comment (acuteenvy) [inline pages/linux/qm-disk-resize.md:3]:
```suggestion
> Resize a virtual machine disk in the Proxmox Virtual Environment (PVE).
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**
